modules/lang-painless/build.gradle

```diff
@@ -40,6 +40,11 @@ opensearchplugin {
   classname 'org.opensearch.painless.PainlessModulePlugin'
 }
 
+ext {
+  // Do not fail on `javadoc` warning (ANTLR generated code)
+  failOnJavadocWarning = false
```

Comment:
@andrross the `lang-painless` is a problem at the moment (I will try to find a better way in https://github.com/opensearch-project/OpenSearch/issues/4541), but TLDR; - ANTLR generated javadocs are full of warnings, we cannot modify these code